src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -50,6 +54,14 @@ interface ResolveReferenceSourceResult {
   identifier: string;
 }
 
+interface DeferConfig {
+  deferToProduction: boolean;
+  favorState: boolean;
+  manifestPathForDeferral: string;
+  manifestPathType?: ManifestPathType;
+  dbt_core_integration_id?: number;
```

Comment:
Usually we use camelcase for internal config. snake case is for python and api interfaces.

Reply:
For clarity here it used as an internal config, not an API or python method.

Reply:
fixed